### PR TITLE
refactor models : all models in centers app

### PR DESCRIPTION
**Pull request by wikibootup**

### CODE REVIEW

File: centers/models/facility.py [hunk 1]
```diff
@@ -1,11 +1,11 @@
 from django.db import models
-from centers.models.center import Center
 
 
 class Facility(models.Model):
+    center = models.ForeignKey('Center')
+
```
Comment (dobestan) [inline centers/models/facility.py]: 오.. 저 이런 변화는 너무 좋은 것 같아요.

Comment (xme1226) [inline centers/models/facility.py]: 오오 리팩토링 감사합니다

File: centers/models/center.py [hunk 2]
```diff
@@ -14,8 +14,7 @@ class Center(models.Model):
     address = models.CharField(max_length=255, blank=True, null=True)
     phone = models.CharField(max_length=45, blank=True, null=True)
     url = models.URLField(blank=True, null=True)
-    hash_id = models.CharField(
-        max_length=12, unique=True)
+    hash_id = models.CharField(max_length=12, unique=True)
```
Comment (darjeeling) [inline centers/models/center.py]: I think hash_id is used in url.
so in that case you need to make it using index

Comment (darjeeling) [inline centers/models/center.py]: and another case 

http://stackoverflow.com/questions/4964197/converting-a-number-base-10-to-base-62-a-za-z0-9

you can try base62 or others for shorten urls

**Pull request merged**